src/navigations/root-stack.js

```diff
@@ -39,22 +36,26 @@ import TopicMemberScreen from '../screens/TopicMemberScreen';
 import TopicPageScreen from '../screens/TopicPageScreen';
 import Topics from '../screens/Topics';
 import VideoViewerScreen from '../screens/VideoViewer';
+import HelpCenter from '../screens/WebView/HelpCenter';
+import PrivacyPolicies from '../screens/WebView/PrivacyPolicies';
 import WhotoFollow from '../screens/WhotoFollow';
 import api from '../service/config';
-import {AddParticipant, ContactScreen, DetailDomainScreen, ProfileScreen} from '../screens';
 import {InitialStartupAtom, LoadingStartupContext} from '../service/initialStartup';
 import {NavigationConstants} from '../utils/constants';
+import HomeBottomTabs from './HomeBottomTabs';
+import KeyboardWrapper from './KeyboardWrapper';
+import OneSignalNavigator from './OneSignalNavigator';
+
 import {followersOrFollowingAtom} from '../screens/ChannelListScreen/model/followersOrFollowingAtom';
 import {useInitialStartup} from '../hooks/useInitialStartup';
 import {COLORS} from '../utils/theme';
 
 const RootStack = createNativeStackNavigator();
 
-export const RootNavigator = ({currentScreen}) => {
+export const RootNavigator = () => {
   const initialStartup = useRecoilValue(InitialStartupAtom);
   const [following, setFollowing] = useRecoilState(followersOrFollowingAtom);
   const loadingStartup = useInitialStartup();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `followersOrFollowingAtom` and `useInitialStartup` are duplicated. They were already imported at the beginning of the file. Please remove these duplicate imports to keep the code clean and maintainable.
```diff
- 49:  import {followersOrFollowingAtom} from '../screens/ChannelListScreen/model/followersOrFollowingAtom';
- 50:  import {useInitialStartup} from '../hooks/useInitialStartup';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->